### DISCUSSION TITLE
Unable to Extract Tabular Data from a streamed PDF

### DISCUSSION THREAD

**Discussion by ertanuj96** [jsvine/pdfplumber#630] [Ask for help with specific PDFs]
Hello, 
I am unable to extract tablular data from this pdf. Please refer to the attached screenshot for code used.
![Screenshot 2022-03-24 124355](https://user-images.githubusercontent.com/27552153/159862365-0c6cf156-68cc-4179-ac96-961e873d75b5.png)

Please help , I am getting None when trying to print Table on the cmd . I have attached the PDF as well.
Please have a look & let me know if pdfplumber can help me to extract data out of this PDF.

[hollywood-march-revenue.pdf](https://github.com/jsvine/pdfplumber/files/8339196/hollywood-march-revenue.pdf)

Thanks

**Comment by jsvine** [comment]
Hi @ertanuj96, and thanks for your interest in this library. In this PDF, the table in question does not have any explicit vertical lines separating the columns.  I would recommend:

- Reading the [documentation re. table-extraction settings](https://github.com/jsvine/pdfplumber#table-extraction-settings), if you have not already done so. Two settings you might particularly want to examine are `"vertical_strategy": "text"` and `"explicit_vertical_lines": [...]`.

- Examining this discussion / response for an approach to identifying a table's missing left- and right-side edges: https://github.com/jsvine/pdfplumber/discussions/621#discussioncomment-2335331

- Using `page.to_image().debug_tablefinder(...)` to examine the effect of various table-extraction settings.

Let us know what progress you make, and whether you still have further questions after that.
